lib/authorize.js

```diff
@@ -0,0 +1,55 @@
+import { getLogger } from 'appium-logger';
+import { exec } from 'teen_process';
+import { getPath } from 'appium-xcode';
+import path from 'path';
+
+export async function authorize(insecure) {
+
+  let logger = getLogger('authorize');
```

Comment:
I agree. What I've been doing is putting:

```js
const logger = getLogger('authorize');
```

up below the imports.

And can we call this logger 'AuthorizeIOS'?